### PR TITLE
Extended Support to Interconnect Resources for API2200

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended Support to below resources for API 2200

Interconnect
Interconnect Types
Logical Interconnect
Logical Interconnect Group

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by yuvirani — APPROVED** [review]

File: examples/logical_interconnects.tf [hunk 1]
```diff
@@ -1,37 +0,0 @@
-provider "oneview" {
```
Comment (VenkateshRavula) [inline examples/logical_interconnects.tf:1]: why did you delete this..? Is it by mistake..?

Comment (AsisBagga) [reply examples/logical_interconnects.tf:1]: 2 example files of LI was unnecessarily created, in incorporated to one. check LI example file.

**Review by VenkateshRavula — APPROVED** [review]
